packages/ember-glimmer/tests/integration/helpers/log-test.js

```diff
@@ -0,0 +1,58 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import Logger from 'ember-metal/logger';
+
+moduleFor('Helpers test: {{log}}', class extends RenderingTest {
+
+  constructor(assert) {
+    super(...arguments);
+
+    this.assert = assert;
```

Comment:
I would just do this in the super (super super super) class. I think having `this.assert` just makes sense for all the subclasses.